### DISCUSSION TITLE
Tele Yucatán (MX)

### DISCUSSION THREAD

**Discussion by GiovanniSebastian** [iptv-org/community#930] [Q&A]
### Channel ID

TeleYucatan.mx

### Stream URL (optional)

http://www.teleyucatan.gob.mx/en_vivo

### Notes (optional)

With Token

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Sphinxroot** [comment]
streamlink pipe it http://conceptoweb-studio.com/radio/video/trecevision/

and send to kodi addon  also pipe it 

other than that its a token  10m + IP hash

**Comment by BellezaEmporium** [comment]
http://mc-qs.lanesh4d0w.tech/iptv-query?streaming-ip=http://conceptoweb-studio.com/radio/video/trecevision/

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19855
